lib/Serialization/Deserialization.cpp

```diff
@@ -1611,6 +1611,13 @@ ModuleFile::resolveCrossReference(ModuleID MID, uint32_t pathLen) {
                                            getXRefDeclNameForError());
       }
 
+      if (memberName.getKind() == DeclBaseName::Kind::Destructor) {
+        assert(isa<ClassDecl>(nominal));
+        // Force creation of an implicit destructor
+        auto CD = dyn_cast<ClassDecl>(nominal);
+        (void) CD->getDestructor();
```

Comment:
How about just adding the destructor to the values vector and skipping the second lookup below?

Reply:
Thanks! I updated.